docs/topics/db/transactions.txt

```diff
@@ -252,6 +252,130 @@ by Django or by third-party libraries. Thus, this is best used in situations
 where you want to run your own transaction-controlling middleware or do
 something really strange.
 
+Performing actions after commit
+===============================
+
+.. versionadded:: 1.9
+
+Sometimes you need to fire off an action related to the current database
+transaction, but only if the transaction successfully commits. Examples: a
```

Comment:
Eh, never mind re Celery. It's used several other places as an example, and I think that's fine.